test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperK8sService.java

```diff
@@ -193,7 +197,7 @@ private V1beta1ClusterRole getClusterRole() {
 
     private V1Deployment getDeployment() {
         V1Container container = new V1ContainerBuilder().withName("zookeeper-operator")
-                                                        .withImage("pravega/zookeeper-operator:latest")
+                                                        .withImage(ZOOKEEPER_OPERATOR_IMAGE)
```

Comment:
are we planning to have snapshot builds for zookeeper operator?

Reply:
I think it's interesting to make the Zookeeper operator image configurable in case there's some issue with the `latest` operator or something.